examples/platform/silabs/SiWx917/SiWx917/rsi_if.c

```diff
@@ -561,6 +562,10 @@ void wfx_rsi_task(void * arg)
                 /* Checks if the assigned IPv6 address is preferred by evaluating
                  * the first block of IPv6 address ( block 0)
                  */
+                if (!hasNotifiedIPV6)
```

Comment:
The comment above should be moved to the if() statement it applies to. Similar change also needed below.